docs/getting_started.md

```
@@ -86,44 +97,37 @@ E   ModuleNotFoundError: No module named 'ansible_collections'
 
-Add a `test_integration.py` file to the `tests/integration` directory of the
-ansible collection:
+Prefer the Molecule CLI (and tox-ansible for collections):
 
-```python
-"""Tests for molecule scenarios."""
-
-from __future__ import absolute_import, division, print_function
-
-from pytest_ansible.molecule import MoleculeScenario
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)